### PR TITLE
Add ChannelKinds needed by IP Camera Adapter

### PR DESCRIPTION
**Pull request by dhylands**
r? @Yoric

### CODE REVIEW

File: src/services.rs [hunk 1]
```diff
@@ -195,6 +195,72 @@ pub enum ChannelKind {
...
+    /// use foxbox_taxonomy::services::*;
+    /// use foxbox_taxonomy::parse::*;
+    ///
+    /// let parsed = ChannelKind::from_str("\"Name\"").unwrap();
+    /// assert_eq!(parsed, ChannelKind::Name);
+    /// ```
+    Name,
+
+    /// The service is used to read or set the location associated with the
+    /// service. This is typically used to store the room within the house
```
Comment (Yoric) [inline src/services.rs]: So far, we discussed the idea of putting this in a `tag`. What's the use case of putting it somewhere else?

**Comment by Yoric** [comment]
r=me, but I'm not sure `Location` is really useful.

**Comment by dhylands** [comment]
Perhaps Name and Location could both be tags. Are tags persisted? Open ZWave persists both of these things and when you have lots of "things" both of those items are extremely useful, but I could see both of them being implemented as tags.

**Comment by Yoric** [comment]
Persistence of tags is in https://github.com/fxbox/taxonomy/pull/64, so they will be persisted very soon.

If `Name` is not used by the device itself, yeah, I agree that it should rather be a tag.

**Comment by dhylands** [comment]
Rebased and removed Name and Location ChannelKinds, since these would be better dealt with by using tags.

**Pull request merged**